client/landing/gutenboarding/onboarding-block/plans/index.tsx

```diff
@@ -5,11 +5,11 @@ import React, { useEffect, useRef, useState } from 'react';
 import { useI18n } from '@automattic/react-i18n';
 import { Button } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
+import { PlansGrid } from '@automattic/plans-ui';
```

Comment:
Let's make `PlansGrid` the default export of this package and name the package `plans-grid`. After we're about to isolate a stand-alone UI component and not a collection of them.